src/wiki/models/__init__.py

```diff
@@ -99,15 +101,9 @@ def reverse(*args, **kwargs):
 urlresolvers.reverse_lazy = reverse_lazy
 
 # Patch up other locations of the reverse function
-try:
-    from django.urls import base
-    from django import urls
-    from django import shortcuts
-    base.reverse = reverse
-    base.reverse_lazy = reverse_lazy
-    urls.reverse = reverse
-    urls.reverse_lazy = reverse_lazy
-    shortcuts.reverse = reverse
-    urls.reverse_lazy = reverse_lazy
-except ImportError:
-    pass
+base.reverse = reverse
+base.reverse_lazy = reverse_lazy
+urls.reverse = reverse
+urls.reverse_lazy = reverse_lazy
+shortcuts.reverse = reverse
+urls.reverse_lazy = reverse_lazy
```

Comment:
dup of 107 (took me about 27 days to notice it :smile_cat: )

Reply:
OK. I just very quickly looked at the `ImportError`.